tests/srv6/test_srv6_basic_sanity.py

```diff
@@ -0,0 +1,123 @@
+import time
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import wait_until
+
+from srv6_utils import announce_route
+from srv6_utils import find_node_interfaces
+from srv6_utils import check_bgp_neighbors
+from srv6_utils import check_bgp_neighbors_func
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,  # Disable automatic loganalyzer, since we use it for the test
+    pytest.mark.topology("any"),
+    pytest.mark.sanity_check(skip_sanity=True),
```

Comment:
Sanity check is an important feature to ensure that a test script is executed on testbed with good health. It would be better not to hard disable it in code. Otherwise, when a script failed, we do not know if it is testbed issue or test issue.
For debugging purposes, we always can use pytest argument `--skip_sanity` to temporarily skip sanity check.

Reply:
Current dut sanity check is based on minigraph configuration file. These checks are not suitable for the sanity testcases we would like to use. Therefore, I skip these default checks to avoid the following failures which are not applicable to this topology. 

E               {"changed": false, "failed": true, "msg": "Error reading file '/etc/sonic/minigraph.xml': failed to load external entity \"/etc/sonic/minigraph.xml\"\nTraceback (most recent call last):\n  File \"/tmp/ansible_minigraph_facts_payload_q2bjukw_/ansible_minigraph_facts_payload.zip/ansible/modules/minigraph_facts.py\", line 924, in main\n  File \"/tmp/ansible_minigraph_facts_payload_q2bjukw_/ansible_minigraph_facts_payload.zip/ansible/modules/minigraph_facts.py\", line 653, in parse_xml\n  File \"/tmp/ansible_minigraph_facts_payload_q2bjukw_/ansible_minigraph_facts_payload.zip/ansible/modules/minigraph_facts.py\", line 608, in reconcile_mini_graph_locations\n  File \"src/lxml/etree.pyx\", line 3538, in lxml.etree.parse\n  File \"src/lxml/parser.pxi\", line 1876, in lxml.etree._parseDocument\n  File \"src/lxml/parser.pxi\", line 1902, in lxml.etree._parseDocumentFromURL\n  File \"src/lxml/parser.pxi\", line 1805, in lxml.etree._parseDocFromFile\n  File \"src/lxml/parser.pxi\", line 1177, in lxml.etree._BaseParser._parseDocFromFile\n  File \"src/lxml/parser.pxi\", line 615, in lxml.etree._ParserContext._handleParseResultDoc\n  File \"src/lxml/parser.pxi\", line 725, in lxml.etree._handleParseResult\n  File \"src/lxml/parser.pxi\", line 652, in lxml.etree._raiseParseError\nOSError: Error reading file '/etc/sonic/minigraph.xml': failed to load external entity \"/etc/sonic/minigraph.xml\"\n"}

Reply:
How about use pytest command line argument `--skip_sanity` to skip sanity check?
The failure should can be addressed in separate PRs. Like making the `minigraph_facts` module more robust when minigraph.xml file does not exist.

Reply:
Sounds good. Let me remove it and add this option in my ptytest run script.